backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -930,6 +930,10 @@ class ContentDigestSchema(marshmallow.Schema):
                     'for sub-contents. Default is True. '
                     'In first version of the API, this field is always True',
     )
+    modified = marshmallow.fields.DateTime(
+        format=DATETIME_FORMAT,
+        description='date of last modification of content',
```

Comment:
Insteand of 'date of last modification of content', i propose 'date of last modification of content. note: this does not include comments or any subcontents.'

Reply:
by the way, if you add modified, its a good idea to also add created